python/cudf/dataframe/columnops.py

```diff
@@ -92,6 +92,17 @@ def fillna(self, fill_value):
         raise NotImplementedError
 
 
+def column_empty(data_size, dtype, masked):
+    """Allocate a new column like the given *column*
+    """
+    data = rmm.device_array(shape=data_size, dtype=dtype)
+    params = dict(data=Buffer(data))
+    if masked:
+        mask = utils.make_mask(data.size)
+        params.update(dict(mask=Buffer(mask), null_count=data.size))
+    return Column(**params)
```

Comment:
If you want a masked column you should explicitly set the null_count to nonzero and likely fill the mask with zeros. Otherwise, I believe the column constructor will intentionally throw away the validity mask for columns with no nulls.

Reply:
Is there best way to fill the mask with 0s?

Reply:
This will work `cudautils.fill_value(mask, 0)`